### PR TITLE
feat:added notifications

### PR DESCRIPTION
**Pull request by borismarkovsv73**

### CODE REVIEW

File: src/main/java/booking_app_team_2/bookie/controller/NotificationController.java [hunk 1]
```diff
@@ -1,27 +1,53 @@
...
+    @PreAuthorize("hasAnyAuthority('Guest', 'Owner', 'Admin')")
+    public ResponseEntity<NotificationDTO> createNotification(@RequestBody NotificationDTO notification){
+        Notification newNotification = new Notification(notification);
+        Optional<User> userOptional = userService.findOne(notification.getReceiverId());
+        userOptional.ifPresent(newNotification::setReceiver);
+        notificationService.addNotification(newNotification);
+        return new ResponseEntity<>(notification,HttpStatus.CREATED);
     }
 
     //Request param pogledati kod darkovog
```
Comment (GojkoVuckovic) [inline src/main/java/booking_app_team_2/bookie/controller/NotificationController.java:41]: delete comment

File: src/main/java/booking_app_team_2/bookie/service/NotificationServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    }
+
+    @Override
+    public void addNotification(Notification notification) {
+        this.notificationRepository.save(notification);
+    }
+
+    @Override
+    public List<Notification> findByReciever(User user) {
```
Comment (GojkoVuckovic) [inline src/main/java/booking_app_team_2/bookie/service/NotificationServiceImpl.java:56]: There are no validations whether the user wants to recieve those notifications

Comment (borismarkovsv73) [reply src/main/java/booking_app_team_2/bookie/service/NotificationServiceImpl.java:56]: I am aware, I couldn't implement it properly so there is no validation here

**Review by darkokos — APPROVED** [review]
I have nothing to note, aside from missing validation

**Review by GojkoVuckovic — APPROVED** [review]

**Pull request merged**